code/datums/storage/storage.dm

```diff
@@ -4,6 +4,15 @@
  * The parent and real_location variables are both weakrefs, so they must be resolved before they can be used.
  * If you're looking to create custom storage type behaviors, check ../subtypes
  */
+
+#define STORAGE_CELL_INDEX 1
+#define STORAGE_TOP_LEFT_CORNER_INDEX 2
+#define STORAGE_TOP_RIGHT_CORNER_INDEX 3
+#define STORAGE_BOTTOM_LEFT_CORNER_INDEX 4
+#define STORAGE_BOTTOM_RIGHT_CORNER_INDEX 5
+#define STORAGE_LEFT_ROWJOIN_INDEX 6
+#define STORAGE_RIGHT_ROWJOIN_INDEX 7
```

Comment:
This is very bad vibes

Reply:
What exactly should I do about this? Make a separate assoc list for every single element?